libp2p/Common.cpp

```diff
@@ -13,7 +13,6 @@ namespace p2p
 using namespace std;
 
 const unsigned c_protocolVersion = 4;
-static_assert(c_protocolVersion == 4, "Replace v3 compatbility with v4 compatibility before updating network version.");
 
 const NodeIPEndpoint UnspecifiedNodeIPEndpoint = NodeIPEndpoint{{}, 0, 0};
```

Comment:
Constexpr?

Reply:
Not possible currently. It would require constexpr constructor of `NodeIPEndpoint` which would require constexpr constructor of `boost::address` which is not available (not even in the current development version). Good idea for Boost contribution.